packages/plugin-ext/src/main/node/plugin-deployer-impl.ts

```diff
@@ -112,6 +112,10 @@ export class PluginDeployerImpl implements PluginDeployer {
     }
 
     public async deploy(pluginEntry: string): Promise<void> {
+        if (this.pluginDeployerEntries.some(value => value.id() === pluginEntry)) {
+            return Promise.reject(new Error(`Plugin ${pluginEntry} already deployed`));
```

Comment:
I would try now, but suspect that second window will get an exception instead of loaded extension. Should now we just skip here.

btw how about to move filtering out plugins to deploy in `deployMultipleEntries`, so it works for any client who call it, not only for `deploy` method?

Reply:
<img width="1106" alt="Screen Shot 2019-03-25 at 15 44 25" src="https://user-images.githubusercontent.com/3082655/54928876-f16e7900-4f14-11e9-8256-ee66aa063620.png">

A user just wants to load this extension, i don't think we need to throw but don't do anything on backend if it is already deployed and load it on the frontend. Otherwise a user can fix it only by reloading a browser.